google-cloud-tools-plugin/src/com/google/gct/idea/CloudToolsPluginInitializationComponent.java

```diff
@@ -50,6 +52,9 @@ public void initComponent() {
           .registerExtension(
               ConfigurationType.CONFIGURATION_TYPE_EP, new CloudDebugConfigType());
     }
+    if (pluginInfoService.shouldEnable(GctFeature.MANAGEDVM)) {
+      pluginConfigurationService.registerExtension(ServerType.EP_NAME, new ManagedVmCloudType());
```

Comment:
Aha, all it required in the plugin.xml was the one server type registration, so when I tested and saw the MVM Cloud show up in the settings, I assumed I was good to go.

I'll check this out.